transmogrifier/sources/xml/marc.py

```diff
@@ -850,6 +650,104 @@ def _get_holdings_electronic_items(
                     note=holding_note,
                 )
 
+    @classmethod
+    def get_identifiers(cls, source_record: Tag) -> list[timdex.Identifier] | None:
+        identifiers = []
+        identifier_marc_fields = [
+            {
+                "tag": "010",
+                "subfields": "a",
+                "kind": "LCCN",
+            },
+            {
+                "tag": "020",
+                "subfields": "aq",
+                "kind": "ISBN",
+            },
+            {
+                "tag": "022",
+                "subfields": "a",
+                "kind": "ISSN",
+            },
+            {
+                "tag": "024",
+                "subfields": "aq2",
+                "kind": "Other Identifier",
+            },
+            {
+                "tag": "035",
+                "subfields": "a",
+                "kind": "OCLC Number",
+            },
+        ]
+        for identifier_marc_field in identifier_marc_fields:
```

Comment:
Just noting again, could be a good candidate for list of tuples + tuple unpacking in `for` loop.  Don't recall where we landed on the last one, but whatever we did, I'd propose this aligns with it.

Actually, just checked, and it looks like we're sticking with the list of dictionaries.  I'm good with that for now, as long as we're consistent (which this appears to be).